pages/index.vue

```diff
@@ -96,11 +98,11 @@ export default class Index extends Vue {
 
   @spotify.Action('fetchCurrentSong') public fetchCurrentSong!: Function;
 
-  public get showLogin() {
```

Comment:
Is there any reason to remove `public` from getters and methods?

Reply:
I think is not necessary due the fact that is code execute inside the component.

The question for me is, why did you make the method public?

Reply:
I supposed that is needed to be able to access it from the component, but thanks for the information.